dashboard/app/views/home/index.html.haml

```diff
@@ -5,6 +5,10 @@
 
 - if current_user && current_user.teacher? && !current_user.accepted_latest_terms?
   = render partial: 'layouts/terms_interstitial'
+- if current_user && current_user.show_race_interstitial?
+  - current_user.race_interstitial_shown = true
+  - current_user.save
```

Comment:
Yes, you need to call [`#save`](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-save) after modifying attributes on an existing record to persist them, or alternatively you can call [`#update`](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update) and pass the new value to persist directly- see the [Active Record Basics](http://guides.rubyonrails.org/active_record_basics.html) Rails guide, specifically the [CRUD#Update](http://guides.rubyonrails.org/active_record_basics.html#update) section.